src/+otp/+utils/+movie/+recorder/VideoRecorder.m

```diff
@@ -1,11 +1,28 @@
 classdef (Abstract) VideoRecorder < handle    
-    methods (Abstract)
-        frameRate = getFrameRate(obj);
-        setFrameRate(obj, newFrameRate);
-        start(obj, totalFrames);
-        recordFrame(obj, fig);
-        stop(obj);
-        h = play(obj);
+    methods
+        function frameRate = getFrameRate(obj)
+            error('Abstract method getFrameRate must be implemented by a subclass');
```

Comment:
I assume this is an octave workaround. That's fine.

Reply:
Yes.  This emulates an abstract method.